<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>minTab</class>
 <widget class="QDialog" name="minTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>530</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QTreeWidget" name="minTree">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Ubuntu&quot;;</string>
         </property>
         <attribute name="headerDefaultSectionSize">
          <number>250</number>
         </attribute>
         <column>
          <property name="text">
           <string>Value</string>
          </property>
          <property name="textAlignment">
           <set>AlignJustify|AlignVCenter</set>
          </property>
          <property name="foreground">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Parameters</string>
          </property>
          <property name="textAlignment">
           <set>AlignLeft|AlignVCenter</set>
          </property>
         </column>
         <item>
          <property name="text">
           <string>Minimizer parameters</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string>Steps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string>First initial step for line minimizer</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string>Max initial step for line minimizer</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string>Gradient reduction factor for line minimizer</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>Nonbonded interactions</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string>Exclusion for atoms with distances</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string>Scaling factor for 1-4 interactions</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string>Cuttoff distance</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string>Use switching Function</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string>Distance to activate switching function</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string>Distance for inclusion in pair lists</string>
           </property>
          </item>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="resetButton">
           <property name="text">
            <string>Reset to Defaults</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="minButton">
           <property name="text">
            <string>Run</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stopButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
